test-cases/gutenberg/writing-flow/readme.md

```diff
@@ -28,5 +22,3 @@
 - [ ] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
   - [ ] Paragraph
   - [ ] Heading
-#### Undo / Redo - Test Cases
-- [ ] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text
```

Comment:
Good catch! Seems we forgot to remove this in https://github.com/wordpress-mobile/test-cases/pull/147 😅 .